<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1909</width>
    <height>1244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>1746</width>
      <height>678</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_domanda">
               <property name="text">
                <string>Stai cercando qualcosa?</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QComboBox" name="comboBox_scelta">
                 <item>
                  <property name="text">
                   <string>Scuola</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Autore</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Soggetto/Titolo</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Data</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sala</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLineEdit" name="lineEdit_scelta"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QPushButton" name="pushButton_cerca">
                 <property name="text">
                  <string>Cerca</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <column>
              <property name="text">
               <string>Scuola</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Autore</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Soggetto/Titolo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Data</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Sala</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_19">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&lt;&lt;&lt;Nota: un nuovo dipinto viene aggiunto IN BASSO della tabella!&gt;&gt;&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_15">
                     <item>
                      <layout class="QGridLayout" name="gridLayout_2">
                       <item row="0" column="4">
                        <widget class="QLabel" name="label_sala">
                         <property name="text">
                          <string>Sala</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="2">
                        <widget class="QLabel" name="label_titolo">
                         <property name="text">
                          <string>Soggetto/Titolo</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="3">
                        <widget class="QLabel" name="label_data">
                         <property name="text">
                          <string>Data</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="4">
                        <widget class="QLineEdit" name="lineEdit_sala_aggiungi"/>
                       </item>
                       <item row="1" column="1">
                        <widget class="QLineEdit" name="lineEdit_autore_aggiungi"/>
                       </item>
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_scuola">
                         <property name="text">
                          <string>Scuola</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="3">
                        <widget class="QLineEdit" name="lineEdit_data_aggiungi"/>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLabel" name="label_autore">
                         <property name="text">
                          <string>Autore</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="lineEdit_scuola_aggiungi"/>
                       </item>
                       <item row="1" column="2">
                        <widget class="QLineEdit" name="lineEdit_titolo_aggiungi"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                  <widget class="QPushButton" name="pushButton_aggiungi">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>1</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::WheelFocus</enum>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Aggiungi un nuovo dipinto</string>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&lt;&lt;&lt;Nota: prima bisogna schiacciare sulla riga che si vuole eliminare e solo poi schiacciare sul bottone&gt;&gt;&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="pushButton_elimina">
                 <property name="text">
                  <string>Elimina un specifico dipinto</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_18">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>&lt;&lt;&lt;Nota: schiccia per ripristinare la tabella iniziale&gt;&gt;&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_iniziale">
               <property name="text">
                <string>Tabella iniziale</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1909</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>Grafici</string>
    </property>
    <addaction name="actionPercentuale_dipinti_per_Scuola"/>
    <addaction name="actionNumero_dipinti_per_Data"/>
   </widget>
   <addaction name="menuQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionPercentuale_dipinti_per_Scuola">
   <property name="text">
    <string>Percentuale dipinti per Scuola</string>
   </property>
  </action>
  <action name="actionNumero_dipinti_per_Data">
   <property name="text">
    <string>Numero dipinti per Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
